pkg/config/registry/version_override.go

```diff
@@ -48,7 +52,8 @@ func (pkgInfo *PackageInfo) SetVersion(v string) (*PackageInfo, error) {
 		}
 		a, err := expr.EvaluateVersionConstraints(vo.VersionConstraints, v, sv)
 		if err != nil {
-			return nil, err //nolint:wrapcheck
+			logerr.WithError(logE, err).Debug("evaluate the version_constraint")
```

Comment:
If it fails to evaluate `version_constraint`, output debug log and the evaluation result is treated as `false`.